src/components/Pelaporan/ImportForm.vue

```diff
@@ -49,20 +49,27 @@
             <v-row v-if="selectedFile" justify="end" class="mr-0">
               <v-btn large color="success" :loading="isLoading" @click="onImport">{{ $t('label.import') }}</v-btn>
             </v-row>
-            <v-row v-if="errorList.length > 0" class="ml-0">{{ $t('label.error_message') }}</v-row>
-            <v-row
-              v-for="row in errorList"
-              :key="row.rowNumber"
-              class="my-2 mx-1 pa-3 error-message"
-            >
-              <span class="font-weight-bold">{{ $t('label.data_in_row') }} {{ row.rowNumber }}</span>
-              <span
-                v-for="(item, index) in row.data"
-                :key="item.columnName"
+            <v-row v-if="errorList.length > 0 && !isLoading" class="ml-0">{{ $t('label.error_message') }}</v-row>
+            <div v-if="errorType === 'row' && !isLoading">
+              <v-row
+                v-for="row in errorList"
+                :key="row.rowNumber"
+                class="my-2 mx-1 pa-3 error-message"
               >
-                <span class="ml-1">{{ item.description }}</span><span v-if="index + 1 < row.data.length">,</span>
-              </span>
-            </v-row>
+                <span class="font-weight-bold">{{ $t('label.data_in_row') }} {{ row.rowNumber }}</span>
+                <span
+                  v-for="(item, index) in row.data"
+                  :key="item.columnName"
+                >
+                  <span class="ml-1">{{ item.description }}</span><span v-if="index + 1 < row.data.length">,</span>
```

Comment:
pake tanda kurung `(` mana dulu yg di proses. Biar mudah dibaca. Coba kode ini bisa ga

```suggestion
                  <span class="ml-1">{{ item.description }}</span><span v-if="(index + 1) < row.data.length">,</span>
```

Reply:
udah pak